rasa/core/policies/ted_policy.py

```diff
@@ -1059,13 +1064,25 @@ def _load(
             predict_data_example,
         ) = cls._construct_model_initialization_data(model_utilities["loaded_data"])
 
-        model = cls._load_tf_model(
-            model_utilities,
-            model_data_example,
-            predict_data_example,
-            featurizer,
-            execution_context.is_finetuning,
-        )
+        model = ""
+        
+        if config['use_gpu']:
+            model = cls._load_tf_model(
+                    model_utilities,
+                    model_data_example,
+                    predict_data_example,
+                    featurizer,
+                    execution_context.is_finetuning,
+            )
+        else:
+            with tf.device('/cpu:0'):
+                model = cls._load_tf_model(
+                    model_utilities,
+                    model_data_example,
+                    predict_data_example,
+                    featurizer,
+                    execution_context.is_finetuning,
+                )
```

Comment:
As explained in the previous comment, please avoid code duplication by using a nullcontext:
```suggestion
        with(tf.device('/cpu:0') if config['use_gpu'] else contextlib.nullcontext()):
            model = cls._load_tf_model(
                    model_utilities,
                    model_data_example,
                    predict_data_example,
                    featurizer,
                    execution_context.is_finetuning,
            )
```